src/lib/list/selection-list.ts

```diff
@@ -126,7 +126,9 @@ export class MatListOption extends _MatListOptionMixinBase
 
   /** Whether the option is disabled. */
   @Input()
-  get disabled() { return (this.selectionList && this.selectionList.disabled) || this._disabled; }
+  get disabled() {
+    return this._disabled || this.selectionList && this.selectionList.disabled;
```

Comment:
Add the parentheses around the second condition here? Otherwise it's a little ambiguous.